health/main.py

```diff
@@ -121,17 +119,19 @@ def job():
                      % (time.time() - started_at))
 
 
-def main():
+def main(argv=sys.argv[1:]):
```

Comment:
main() function should not accept any arguments, otherwise you can't use it as entry point